### PR TITLE
Add FNOD Banner feature toggle

### PR DESCRIPTION
**Pull request by meckilabritt-navapbc**
Resolves [CASEFLOW-703](https://vajira.max.gov/browse/CASEFLOW-703)

### Description
Add FNOD Banner Feature Toggle

### Acceptance Criteria
- [ ] Code compiles correctly
- [ ] FNOD Banner appears if feature toggle is enabled
- [ ] Veteran date of death is date of death on appeal object and not today's date 

### Testing Plan
1. Go to Queue and search for Veteran with date of death e.g. 500000000. 
If the date of death does not appear, enter a date of death for the`date_of_death` column for the Veteran in the `veterans` table
2. Verify that the FNOD banner does not appear
3. Enable `fnod_banner` feature toggle
5. Go to Queue and search for Veteran with date of death e.g. 500000000. If the date of death does not appear, enter a date_of_death manually for the Veteran in the `veterans` table
6. Verify that banner appears
7. Verify that date of death is the current day's date

### CODE REVIEW

File: client/app/queue/CaseDetailsView.jsx [hunk 1]
```diff
@@ -118,7 +118,7 @@ export const CaseDetailsView = (props) => {
       {(!modalIsOpen || props.userCanScheduleVirtualHearings) && <UserAlerts />}
       <AppSegment filledBackground>
         <CaseTitle appeal={appeal} />
-        { appeal.veteranAppellantDeceased && <FnodBanner appeal={appeal} /> }
+        { appeal.veteran_appellant_deceased && props.featureToggles.fnod_banner && <FnodBanner appeal={appeal} /> }
```
Comment (bgantick) [inline client/app/queue/CaseDetailsView.jsx:121]: we'll also need to address the fact that the banner should show for for non-veteran appellants too.. i think that can be a separate PR though so we can get the FNOD badge released

Comment (meckilabritt-navapbc) [reply client/app/queue/CaseDetailsView.jsx:121]: Should `appeal.veteran_appellant_deceased` be removed?

Comment (bgantick) [inline client/app/queue/CaseDetailsView.jsx:121]: This should do it:

```suggestion
        { appeal.veteranDateOfDeath && props.featureToggles.fnod_banner && <FnodBanner appeal={appeal} /> }
```

**Review by bgantick — CHANGES_REQUESTED** [review]

**Review by bgantick — APPROVED** [review]
LGTM once that flaky jest test is passing
